### ISSUE TITLE
When nodes are removed , attempt to clean up system containers from the host.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rke#42]
rke version v0.0.4-dev

Steps to reproduce the problem:
Create a cluster with 2 nodes in controlplane  and 1 node each in etcd and worker.
Remove one of the nodes in controlplane.

Even after the node is removed from the cluster , system containers continue to be present in the host.

```
8c9b34e1d7a4        rancher/k8s:v1.8.3-rancher1                                                                                                   "kube-apiserver --ins"   19 minutes ago      Up 19 minutes                           kube-api
aad891bd8af0        rancher/k8s:v1.8.3-rancher2                                                                                                   "kube-proxy --v=2 --h"   2 hours ago         Up 2 hours                              kube-proxy
8de5f4bf533a        rancher/k8s:v1.8.3-rancher2                                                                                                   "kubelet --v=2 --addr"   2 hours ago         Up 2 hours                              kubelet
87473c127186        rancher/k8s:v1.8.3-rancher2                                                                                                   "kube-scheduler --lea"   2 hours ago         Up 2 hours                              scheduler
a9c059b4c9a9        rancher/k8s:v1.8.3-rancher2                                                                                                   "kube-controller-mana"   2 hours ago         Up 2 hours                              kube-controller
```

It would be good to make an attempt to clean up system containers from the host when the hosts are reachable.

**Comment by galal-hussein** [comment]
The fix can be tested with v0.0.5-dev

**Comment by sangeethah** [comment]
rke version v0.0.4-dev

Steps to reproduce the problem:
Create a cluster with 2 nodes in controlplane and 1 node each in etcd and worker.
Remove one of the nodes in controlplane.

After the control node is removed from the cluster , system containers get removed from the host.
Only pods  relating to network continues to be present.

```
root@ip-172-31-37-222:/home/ubuntu# docker ps
CONTAINER ID        IMAGE                                      COMMAND             CREATED             STATUS              PORTS               NAMES
dd66e70d1d36        gcr.io/google_containers/pause-amd64:3.0   "/pause"            About an hour ago   Up About an hour                        k8s_POD_kube-flannel-n89ll_kube-system_643963dc-d3a1-11e7-bf80-0a91b38ccf16_0
```
